modules/cnf-topology-aware-lifecycle-manager-installation-cli.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="installing-topology-aware-lifecycle-manager-using-cli_{context}"]
+= Installing the {cgu-operator-full} by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the {cgu-operator-first}.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the latest version of the {rh-rhacm} Operator.
+* Set up a hub cluster with disconnected registry.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `Subscription` CR:
+.. Define the `Subscription` CR and save the YAML file, for example, `talm-subscription.yaml`:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-topology-aware-lifecycle-manager-subscription
+  namespace: openshift-operators
+spec:
+  channel: "4.10"
```

Comment:
Please change channel to stable. I believe we now plan to include stable channel.